### DISCUSSION TITLE
My account has been banned

### DISCUSSION THREAD

**Discussion by voyager-hang** [gotd/td#1753] [General]
### What version of gotd are you using?

```bash
github.com/gotd/td v0.83.0
```

My account was banned after logging in with a verification code, at first I thought it was a problem with the new account, until then I used the account I had registered for many years, just after I logged in successfully, before I could send a message, I was banned, I think it is not an account problem, and as far as I know, many developers have encountered the same problem, is there a solution.

**Comment by Esiarp8** [comment]
https://github.com/LonamiWebs/Telethon
I'm using this python for logging in and manipulating accounts with no problems, but logging in with this td is banned, it's immediate, for both accounts. Couldn't detect where the abuse was triggered.

**Comment by connorzhuang** [comment]
I have also encountered such a problem. Do you have a good solution to share?

**Comment by Esiarp8** [comment]
> I have also encountered such a problem. Do you have a good solution to share?

I've actually switched to a different and started using it, there are plenty of wrapper libraries for other languages available.

**Comment by divyam234** [comment]
> I have also encountered such a problem. Do you have a good solution to share?

@voyager-hang  @connorzhuang Use official telgram client api credentails and Device config should match as official clients.Also put a rate limit middleware if you are uploading and downloading files rigrously

**Comment by sudosz** [comment]
I have exactly the same error
As soon as you enter the account, that account will be deleted
I set the session and device information completely based on reality, but it still causes problems
I also changed the IP
Nothing worked
Please, someone, help

**Comment by anhnmt** [comment]
+1
